Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    private var localStorage: LocalStorage!
+    private let defaultValueToSave = "Hello, World!".data(using: .utf8)!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        if let bundleIdentifier = Bundle.main.bundleIdentifier {
+            storage.localStorage?.removePersistentDomain(forName: bundleIdentifier)
+        }
+    }
+
+    override func tearDown() {
+        localStorage = nil
+        super.tearDown()
+    }
+
+    func testDeleteMethodExistsInAPI() {
+        localStorage.deleteValue(forKey: "key")
+    }
+
+    func testDeleteValueWithExistingKey() {
+        // given
+        let key = "TestKey"
```

Comment:
nit: 

If the content of the key doesn't affect the test, make it short and simple:
```suggestion
        let key = "key"
```
or
```suggestion
        let key = "A"
```

Reply:
fixed